src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -14,9 +14,12 @@
 # - dotnet-source-build (internal)
 #   - PR: ultralite build
 #   - CI: release/* and internal/release/* only, every batched commit, full build
-#   - CI: main only, every batched commit, lite build
 #   - Schedule: main only, full build
 #
+# - dotnet-source-build-lite (internal)
+#   - PR: release/* and main, lite build

```

Comment:
This seems redundant with https://github.com/dotnet/installer/pull/18520/files#diff-3bbdc81270db98c75c3c53e4ac2c8136f7dcd9c97fbe516b1e4d4737f5a45000R15.  Only one pipeline should trigger for PRs and it should run the ultralite configuration.

Reply:
I guess I should add a comment about that :)